backend/src/apps/slack/commands/command.py

```
@@ -118,13 +118,14 @@ def render_text(self, context) -> str:
         """
         return self.template.render(context)
 
-    def handler(self, ack, command, client):
+    def handler(self, ack, command, client, respond=None):
```

Ruff (0.16.1)

[warning] Unused method argument: `respond`

(ARG002)